answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+
+#### Executive votes
+
+Executive votes are a more common occurrence, and are used to change the state, or "governance variables" of the smart contracts constituting the Maker infrastructure. Typically this means modifying the existing "risk parameters" of smart contracts that manage Collateral Debt Positions (CDPs), the debt instruments used to issue DAI and manage its supply. For example, an executive vote could be held to decide whether or not to raise the "stability fee" (i.e. "interest" paid to MKR holders on loans). Executive votes can also introduce new parameters or smart contracts. For example adding a new collateral type once multi-collateral DAI is launched.
+
+Executive votes are binding. If passed, they are automatically implemented on the blockchain after a 24 hr delay (a measure to protect against hacks or governance attacks). However, in practice, while any Ethereum address can make a proposal and trigger a vote, the concentration of MKR holdings in the Maker Foundation, core team, and associated investors (Polychain, Andreessen Horowitz, Placeholder VC, among others), mean that only executive votes created by the core team have a reasonable chance of passing. For now, proposals for executive votes are created in a more traditional, centralized process within the Maker Foundation, utilizing a formal process that attempts to emulate the scientific process. Feedback from MKR holders and the general "governance community" is taken into account at various stages. Maker's goal is to perform a "gradual decentralization" of this process over time as the system matures.
+
+#### Risk Governance Framework
+
+To protect itself from bias, polarizing debate, and self serving rhetoric, Maker has created an internal process that utilizes an objective "risk governance framework". In this process, "risk teams" (professionals employed by the Maker Foundation) utilize a formal, rigorous framework for continually evaluating the qualitative and quantitative risks associated with various collateral types. For instance the volatility risk, liquidity risk, and stability of the asset fundamentals. The outputs of this framework are then input into well-understood risk models borrowed from traditional finance to determine optimal "risk parameters" such as the debt ceiling, liquidation ratio, stability fee, and other parameters. The core team then presents their new models, data and suggested parameters to MKR holders and the community at large. Feedback from the community is incorporated and then put into a proposal for an executive vote. The executive vote itself can be used to gather further feedback from MKR holders, which can be incorporated back into the proposal. Eventually, MakerDAO intends to further decentralize this process, creating multiple risk teams elected by MKR holders that compete with each other using the risk governance framework, creating a "decentralized, open scientific risk management community".
```

Comment:
Very interesting, but I want to see where this has been occurring.

Reply:
Have done my best to fill this out more. Left a more direct question in the #help channel (below) of Maker's chat a couple hours ago asking about specifically where the conversation happens, have not yet received a reply.

![image](https://user-images.githubusercontent.com/1634777/54064273-a2f87500-41c7-11e9-8cad-b026a2fa109e.png)